domain_models/models.py

```diff
@@ -211,3 +212,11 @@ def get_data(self):
         return dict((name, field.get_builtin_type(self))
                     for name, field in
                     six.iteritems(self.__class__.__fields__))
+
+    def set_data(self, data):
+        """Set dictionary data to model.
+
+        :param dict data:
+        """
+        for name, value in six.iteritems(data):
+            self.__class__.__fields__[name].set_builtin_type(self, value)
```

Comment:
And still, what will happen if I call ``Profile(...).set_data({id: 1, name: 'John', 'undefined': 'abracadabra'})``. If I'm not wrong, it will raise a ``KeyError``. At the same time, I think that we want to see ``AttributeError('Attribute "undefined" does not exist')`` in such cases.